### ISSUE TITLE
[Bug] Interferes with Java Edition Minecraft FPS

### ISSUE THREAD

**Issue by numeralnathan** [xmichelo/Beeftext#462]
#### Expected behavior
Beeftext should not interfere with Minecraft frames per second (FPS) or any other game.

#### Actual behavior
While playing Minecraft, the screen will stutter.  The game is hard to play this way.

#### Steps to reproduce the behavior
Install Minecraft Java Edition.  Start playing the game and hold the move forward button down.  Notice that the screen is stuttering.  After holding the forward button long enough, Beeftext stops interfering and FPS returns to normal.  However, if I let go of the forward button and then press it again Beeftext causes the stuttering again.

#### Beeftext version

12.0

#### Operating system version (e.g Windows 10)

Windows 11 version 21H2 build 22000.434

**Comment by xmichelo** [comment]
Hi.

I installed Minecraft Java Edition from the PC Game Pass subscription and could not reproduce the issue on Windows 10. Everything works flawlessly. Could you please do tell me if the problem occurs when Beeftext is:

- Disabled (File ➜ Pause Beeftext)
- Not running (I guess you already have the answer for that, but just to be sure).

**Comment by numeralnathan** [comment]
I uninstalled Beeftext after submitting this ticket.  I just installed it and I cannot reproduce the problem.  Beeftext is enabled and working.  In fact, I could get it to replace text while typing in Minecraft.

I don't have as many phrases configured as I did before.  I looked through the phrases I had configured and do not have any that rely on the w key (i.e., Minecraft move forward).

I'll leave Beeftext running and see if I can reproduce the problem.  I've set a reminder to come back next week if nothing happens.
